stderr of test 'python3-udf-modules.Bug-6797` in directory 'sql/test/BugTracker-2019` itself:


# 11:24:24 >  
# 11:24:24 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=30931" "--set" "mapi_usock=/var/tmp/mtest-51109/.s.monetdb.30931" "--forcemito" "--dbpath=/home/ferreira/repositories/MonetDB-Nov2019/BUILD/var/MonetDB/mTests_sql_test_BugTracker-2019" "--set" "embedded_py=3" "--set" "embedded_c=true"
# 11:24:24 >  

# builtin opt 	gdk_dbpath = /home/ferreira/repositories/MonetDB-Nov2019/BUILD/var/monetdb5/dbfarm/demo
# builtin opt 	mapi_port = 50000
# builtin opt 	mapi_open = false
# builtin opt 	mapi_ipv6 = false
# builtin opt 	mapi_autosense = false
# builtin opt 	sql_optimizer = default_pipe
# builtin opt 	sql_debug = 0
# cmdline opt 	gdk_nr_threads = 0
# cmdline opt 	mapi_open = true
# cmdline opt 	mapi_port = 30931
# cmdline opt 	mapi_usock = /var/tmp/mtest-51109/.s.monetdb.30931
# cmdline opt 	gdk_dbpath = /home/ferreira/repositories/MonetDB-Nov2019/BUILD/var/MonetDB/mTests_sql_test_BugTracker-2019
# cmdline opt 	embedded_py = 3
# cmdline opt 	embedded_c = true

# 11:24:25 >  
# 11:24:25 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-51109" "--port=30931"
# 11:24:25 >  


# 11:24:25 >  
# 11:24:25 >  "Done."
# 11:24:25 >  

